backend/src/core/annotation_unit.py

```diff
@@ -0,0 +1,38 @@
+""" Class to instantiate Annotation Units and support its functions"""
+
+
+class AnnotationUnit:
+    """ Annotation Unit Class that houses the Genomic Unit and its corresponding dataset """
+
+    def __init__(self, genomic_unit, dataset):
+        self.genomic_unit = genomic_unit
+        self.dataset = dataset
+
+    def get_genomic_unit(self):
+        """Getter method"""
+        return self.genomic_unit['unit']
+
+    def get_dataset(self):
+        """Getter method"""
+        return self.dataset['data_set']
+
+    def get_missing_dependencies(self):
```

Comment:
This method could use a unit test to verify this independently works.  Parts of this is tested elsewhere but now that this could has been pulled into this class, it will be easier to more finely unit test this one method.

Reply:
Pushed this up!